package.json

```diff
@@ -102,6 +102,7 @@
     "@aws-sdk/util-hex-encoding": "^3.47.0",
     "@types/ua-parser-js": "^0.7.35",
     "detect-browser": "^5.2.0",
+    "moment-timezone": "^0.5.42",
```

Comment:
Is there a browser native API over adding a dependency? If not hard enough, we would like to limit the dependencies.

Reply:
Removed the dependency.